README.adoc

```diff
@@ -182,34 +215,40 @@ The [hotspot=build file=0]`<build/>` section gives details of the two plugins th
 to build this project.
 
 * The Maven plugin for generating a WAR file as one of the output files.
-* The Liberty Maven plugin, which gives details of the name, version, and file type of 
-Open Liberty runtime package which it will download from the public Maven repository.
+* The Liberty Maven plugin, which allows you to install applications into Open Liberty and manage the server instances.
```

Comment:
Change
"The Liberty Maven plugin, which allows you to install applications into Open Liberty and manage the server instances."
To
"The Liberty Maven plug-in, which allows you to install applications into Open Liberty and manage the server instances."